### DISCUSSION TITLE
Amazed by how little it costs...

### DISCUSSION THREAD

**Discussion by jakob1379** [karakeep-app/karakeep#741] [Show and tell]
So I migrated all my articles from pocket to hoarder, as I have always wanted the AI tagging. Together with a newly built homelab this was an obvious chance!

![image](https://github.com/user-attachments/assets/822fb75d-83cf-4ae1-9318-e076d77002df)

~5k articles have been processed and this is the price after crawling most of them!

The transferring had to be tweaked as to not make my little pi4b bite the dust. I didn't really like the default pocket import as it does not transfer already archived articles into archive, so I just made a new list “to be archived” which could then be iterated to archive all the articles. The API works fairly well, but allowing for multiple IDs would be great so I do not have to make an awkward printf hack. I might be simply using the api wrong, but this was the only way I could get it at take multiple ids.

```bash
yq -rp csv -o json '.[] | select(.status == "archive") | .url' -r pocket.csv   | xargs -n 5 -P 4 bash -c 'npx @hoarderapp/cli bookmarks add --list-id xd5xdxgxik3rm23itqn4g5x5 $(printf -- "--link %s " "$@")' _
````

**Comment by MohamedBassem** [comment]
@jakob1379 Yeah, the AI tagging became extremely cheap. I think since I started hoarder and with all the development and my family/friends using my instance, I don't think I've exceeded $5 of credit on OpenAI so far which is extremely impressive.

As for respecting the archival status in imports, I'm planning to tackle this in https://github.com/hoarder-app/hoarder/issues/703

**Reply by jakob1379** [reply]
Sweet! Would be nice if it was a flag or just an protected list id you could just specify when using the api
